pkg/console/operator/status.go

```diff
@@ -235,96 +232,6 @@ func (c *consoleOperator) ConditionResourceSyncDegraded(operatorConfig *operator
 	return operatorConfig
 }
 
-func (c *consoleOperator) ConditionsManagementStateUnmanaged(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
```

Comment:
need to take case of https://github.com/openshift/console-operator/blob/master/pkg/console/operator/operator.go#L221

Reply:
+1 got it!